src/common/utils/autocomplete.js

```diff
@@ -32,9 +32,31 @@ function getCountryFromCode(countryCode) {
   return countryFromCode === undefined ? countryCode : countryFromCode.label;
 }
 
+/**
+ * generate list of airport codes from the airport list
+ * @param {Array<Object>}
+ * @returns {Array<String>} List of airport codes
+ */
+function listOfAirportCodes(airportList) {
```

Comment:
When this listOfAirportCodes method will be called? Is it called while the application loads?

Reply:
It is only execute once, when the `autocomplete.js` module is first required.
* after that is not used --> the [require cache](https://nodejs.org/api/modules.html#:~:text=require.cache%20%23&text=Modules%20are%20cached%20in%20this,will%20result%20in%20an%20error.) handles all usage after that I think.
* The below show it being triggered twice, this is because of nodemon [unloads](https://stackoverflow.com/questions/6676612/unloading-code-modules#:~:text=%22Unloading%22%20a%20node%20module%20means,was%20requiring%20%22%2Flib%2F.) from the node module require cache, the `start.js` module, which triggers the autocomplete module again.

<img width="1440" alt="Screenshot 2024-02-16 at 13 08 31" src="https://github.com/UKHomeOffice/egar-public-site-ui/assets/123387957/ca69ab9d-c38a-4237-90b9-4e3677f66251">